release_notes/ocp-4-7-release-notes.adoc

```diff
@@ -186,14 +186,19 @@ If a machine config pool (MCP) is in a `degraded` state, the Machine Config Oper
 === Nodes
 
 [id="ocp-4-7-nodes-descheduler-ga"]
-==== Descheduler and profiles GA
+==== Descheduler is generally available
 
 The descheduler is now generally available. The descheduler provides the ability to evict a running pod so that the pod can be rescheduled onto a more suitable node. You can enable one or more of the following descheduler profiles:
 
 * `AffinityAndTaints`: evicts pods that violate basic pod and node affinity and node taints.
```

Comment:
@bergerhoffer should we say "evicts pods that violate basic pod node affinity & node taints" ? existing one has two 'and's in the sentence so asking.

Reply:
@kasturinarra As I mentioned in the other PR, I reworded this. Let me know if it looks better to you now or not.

Reply:
@bergerhoffer Thanks, looks good to me now.